tests/rptest/scale_tests/partition_balancer_scale_test.py

```diff
@@ -112,15 +112,15 @@ def test_partition_balancer_with_many_partitions(self, type):
             # FIXME: this is not enough data to keep up a background load through
             # the test.
             # https://github.com/redpanda-data/redpanda/issues/6245
-            message_size = 128 * (2 ^ 10)
+            message_size = 128 * (2**10)
```

Comment:
This might address the preceding FIXME comment that references https://github.com/redpanda-data/redpanda/issues/6245

Although it will also make the test quite a bit slower, this probably needs running before merging to check the test is still happy.

Maybe the changes in scale_tests/ should be split out into a separate PR so that they can be tested separately.

Reply:
will separate those changes

Reply:
PR here: https://github.com/redpanda-data/redpanda/pull/8040

Reply:
separated in: https://github.com/redpanda-data/redpanda/pull/8040